taskfile/cmd.go

```diff
@@ -11,8 +11,9 @@ import (
 // Cmd is a task command
 type Cmd struct {
 	Cmd         string
-	Silent      bool
 	Task        string
+	ForEach     *ForEach
```

Comment:
Happy to use `for` instead. It keeps things nice and concise. `foreach` was based off the comments in #82.

Reply:
@andreynering Quick question on this. `foreach: source` kind of made sense because it reads well in English.

- "Do thing for each source"

Now that we have `for` instead of `foreach`, I think it makes more sense to use the attribute name directly rather than the singular form of the noun. i.e. `for: sources` reads better to me than `for: source`.

This would also make it easier to be consistent if we ever loop over other fields. `for: generates` makes more sense than trying to work out what the singular form of `generates` is.